manifests/apache.pp

```diff
@@ -62,8 +62,6 @@
       owner => $::apache::user,
       group => $::certs::group,
       mode  => '0440',
-    } ->
```

Comment:
I agree, if we actually `include ::apache` to get the user and group, we should just leave this in.